File: Feitos em C/1010.c
#include<stdio.h>
void main (){
	int a=0,b=0,d=0,e=0;
	double c=0,f=0;
	scanf("%d",&a);
	scanf("%d",&b);
	scanf("%lf",&c);
	scanf("%d",&d);
	scanf("%d",&e);
	scanf("%lf",&f);
	printf("VALOR A PAGAR: R$ %.2f\n",(b*c)+(e*f));
	return 0;
}

File: Feitos em C Sharp/1010.cs
using System;
using System.Globalization;
namespace ConsoleApp8 {
    class Program {
        static void Main(string[] args) {
            string[] aux = Console.ReadLine().Split();
            int a = int.Parse(aux[0]);
            int b = int.Parse(aux[1]);
            double c = double.Parse(aux[2], CultureInfo.InvariantCulture);
            string[] aux2 = Console.ReadLine().Split();
            int d = int.Parse(aux2[0]);
            int e = int.Parse(aux2[1]);
            double f = double.Parse(aux2[2], CultureInfo.InvariantCulture);
            Console.WriteLine("VALOR A PAGAR: R$ " + ((b * c) + (e * f)).ToString("F2", CultureInfo.InvariantCulture));
        }
    }
}